rubrik_cdm/data_management.py

```diff
@@ -885,3 +885,108 @@ def get_sla_objects(self, sla, object_type, timeout=15):
                         sla, object_type))
 
             return vm_name_id
+
+    def _time_in_range(self, start, end, point_in_time):
+        """Checks if a specific datetime exists in a start and end time. For example:
+        checks if a recovery point exists in the available snapshots
+        
+        Arguments:
+            start {datetime} -- The start time of the recoverable range the database can be mounted from.
+            end {datetime} -- The end time of the recoverable range the database can be mounted from. 
+            point_in_time {datetime} -- The point_in_time you wish to Live Mount.
+
+        Returns:
+            bool -- True if point_in_time is in the range [start, end]."""
+        
+        if start <= end:
+            return start <= point_in_time <= end
+        else:
+            return start <= point_in_time or point_in_time <= end
+
+    def sql_live_mount(self, db_name, date, time, sql_instance=None, sql_host=None, mount_name=None, timeout=30):  # pylint: ignore
+        """Live Mount a database from a specified recovery point. 
+
+        Arguments:
+            db_name {str} -- The name of the database to Live Mount.
+            date {str} -- The recovery_point date you wish to Live Mount formated as `Month-Day-Year` (ex: 1-15-2014). 
+            time {str} -- The recovery_point time you wish to Live Mount formated formated as `Hour:Minute AM/PM` (ex: 1:30 AM).
+
+        Keyword Arguments:
+            sql_instance {str} -- The SQL instance name with the database you wish to Live Mount.
+            sql_host {str} -- The SQL Host of the database/instance to Live Mount.
+            mount_name {str} -- The name given to the Live Mounted database i.e. AdventureWorks_Clone.
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {30})
+
+        Returns:
+            dict -- The full response of `POST /v1/mssql/db/{id}/mount`.
+        """
+        
+        from datetime import datetime
```

Comment:
Move `from datetime import datetime` to the top of the file

Reply:
I have now

Reply:
And updated the vmware and sql sample, copy and paste mistake.